README.md

```diff
@@ -1,10 +1,10 @@
 # wolfCLU
 
-This is the wolfSSL: Command Line Utility (wolfCLU).
+This is the wolfSSL Command Line Utility (wolfCLU).
 
-## wolfSSL Install
+## wolfSSL Installation
 
-To use this feature, please configure and install wolfssl with the following commands:
+Configure and install wolfSSL with the following commands:
 
 ```
 ./configure --enable-pwdbased --enable-opensslextra --enable-keygen --enable-ed25519 --enable-certgen
```

Comment:
maybe also --enable-ed448 as well?

Reply:
@JacobBarthelmeh is that needed?

Reply:
I only suggested it because you have --enable-ed25519 .  Maybe remove --enable-ed25519 ?

Reply:
I think the ed448 is not needed. The hope though is that people will use the --enable-wolfclu. That's what I would list instead for simplicity. Then power users can go in later and pick and choose in something like a future wolfCLU manual.

Reply:
Forgot about `--enable-wolfclu`. I'll modify this to use that instead.